### PR TITLE
Add user activate/deactivate commands

### PR DESCRIPTION
**Pull request by aanil**
Added the following commands with email as required argument.
* dds user activate
* dds user deactivate

### CODE REVIEW

File: dds_cli/__main__.py [hunk 1]
```diff
@@ -490,6 +490,100 @@ def get_info_user(click_ctx, username):
...
+                no_prompt=click_ctx.get("NO_PROMPT", False),
+            ) as manager:
+                if email:
+                    manager.user_activation(email=email, action="reactivate")
+                else:
+                    LOG.error(
+                        "You must either specify the the e-mail "
+                        "address of the user to be deleted"
+                    )
+                    sys.exit(1)
```
Comment (inaod568) [inline dds_cli/__main__.py:528]: Not needed since the `email_arg` is required.

File: dds_cli/__main__.py [hunk 2]
```diff
@@ -490,6 +490,100 @@ def get_info_user(click_ctx, username):
...
+                no_prompt=click_ctx.get("NO_PROMPT", False),
+            ) as manager:
+                if email:
+                    manager.user_activation(email=email, action="deactivate")
+                else:
+                    LOG.error(
+                        "You must either specify the the e-mail "
+                        "address of the user to be deleted"
+                    )
+                    sys.exit(1)
```
Comment (inaod568) [inline dds_cli/__main__.py:575]: Same here

Comment (aanil) [reply dds_cli/__main__.py]: Done!

File: dds_cli/__main__.py (hunk 1 above)
Comment (aanil) [reply dds_cli/__main__.py]: Done!

**Review by inaod568 — APPROVED** [review]

**Pull request merged**